### ISSUE TITLE
Have a fully created app as reference

### ISSUE THREAD

**Issue by tnguyen14** [choojs/create-choo-app#58]
I think it would be useful to have a completely set up project from scratch using `create-choo-app` included in this repo as a reference. Sometimes, I want to see what a conventionally set up choo app would look like, and cherry pick some features that would make sense for me. I think this is most applicable to an existing project.

**Comment by chrisdwheatley** [comment]
I like the idea of having a reference for those interested in seeing how `create-choo-app` structures things.

We could easily add an example under an `example` directory in this repo and call it out in the readme, my only concern would be ensuring it's kept up to date as things change.

Am I along the right lines with what you were suggesting @tnguyen14?

**Comment by tnguyen14** [comment]
Yes, that's exactly right.

I agree that it should be kept up to date as things change. Along that line, perhaps we can have a script to automatically update that directory with the latest example when changes are made?

**Comment by chrisdwheatley** [comment]
It might be worth referencing the example in the main Choo repo (https://github.com/choojs/choo/tree/master/example) and then ensuring that `create-choo-app` represents the same structure.

**Comment by tnguyen14** [comment]
Yeah, I agree with that. Need to consider one as the source of truth, and update the other. In this case, it might be easier to consider `create-choo-app` to be that source, and as part of the build process, somehow submit a PR against the main `choo` repo to update the `example` directory?

**Comment by ungoldman** [comment]
Might be easier to just link to the single source of truth rather than maintaining a copy.
